### PR TITLE
LLT-5068 Bump Moose 4.0.1 backport

### PR DESCRIPTION
**Pull request by matislovas**
Bumping Moose to v4.0.1

### :ballot_box_with_check: Definition of Done checklist
- [x] Commit history is clean ([requirements](../blob/main/docs/git_commit_messages_requirements.md))
- [x] README.md is updated
- [x] changelog.md is updated
- [x] Functionality is covered by unit or integration tests

### CODE REVIEW

**Comment by packgron** [comment]
Use luis commit for CVE fix, to keep in line with main

File: Cargo.toml [hunk 1]
```diff
@@ -138,7 +138,7 @@ rand = "0.8"
 rstest = "0.11.0"
 rupnp = { version = "1.1.0", default-features = false }
 rustc-hash = "1"
-serde = { version = "1.0", features = ["derive"] }
+serde = { version = "=1.0.197", features = ["derive"] }
```
Comment (matszczygiel) [inline Cargo.toml:141]: Why?

Comment (matislovas) [reply Cargo.toml:141]: `cargo` fails to select `serde` versions.

**Comment by matislovas** [comment]
> Use luis commit for CVE fix, to keep in line with main

Done

**Comment by matszczygiel** [comment]
+1

File: Cargo.toml (hunk 1 above)
Comment (jjanowsk) [reply Cargo.toml:141]: Why not just call `cargo update -p serde`? Or putting here `>=1.0.197` at least? I'm worried `=1.0.197` can cause additional problems next time some other dependency updates and requires newer version

Comment (matislovas) [reply Cargo.toml]: Yeah, good idea, I just blindly followed cargo's suggestion :smile:  fixed.

**Review by jjanowsk — APPROVED** [review]
+1

**Pull request merged**